modules/selecting-cluster-capabilities.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// *installing/cluster-capabilities.adoc
+
+:_content-type: PROCEDURE
+[id="selecting-cluster-capabilities_{context}"]
+= Selecting cluster capabilities
+You can select cluster capabilities by following one of the installation methods that allow you to customize your cluster, such as "Installing a cluster on AWS with customizations" or "Installing a cluster on GCP with customizations". 
+
+During a customized installation, you create an `install-config.yaml` file that contains the configuration parameters for your cluster. You can use the following configuration parameters to select cluster capabilities:
+
+[source,yaml]
+----
+capabilities:
+  baselineCapabilitySet: vCurrent <1>
+  additionalEnabledCapabilities: <2>
+  - openshift-samples
```

Comment:
Hmm, is it a good example? I mean it technically works but when we set baseline to `vCurrent`, setting additionalEnabledCapabilities seems to be redundant. WDYT?

Reply:
+1, refer to good example in [openshift-docs#52341](https://github.com/openshift/openshift-docs/pull/52341/files#diff-a9762e41a395b4426649be01574f780133fafd2472bb5fb0002f227985232565R112-R118) where wking suggested, it might make more sense?

Reply:
Good point, I will use the example from that PR (except I will include Console and exclude Insights in the example because if people copy and paste it, it could be a problem if they install without a web console)